index.rst

```diff
@@ -28,19 +28,19 @@ Welcome! This is the documentation for Numpy and Scipy.
   <table class="contentstable" align="center"><tr>
     <td width="50%">
       <p class="biglink"><a class="biglink" href="numpy/">Complete Numpy Manual</a><br/>
-        <span><a href="numpy/numpy-html-1.17.0.zip">[HTML+zip]</a></span>
+        <span><a href="numpy/numpy-html.zip">[HTML+zip]</a></span>
       </p>
       <p class="biglink"><a class="biglink" href="numpy/reference/">Numpy Reference Guide</a><br/>
-        <span><a href="numpy/numpy-ref-1.17.0.pdf">[PDF]</a></span>
+        <span><a href="numpy/numpy-ref.pdf">[PDF]</a></span>
       </p>
       <p class="biglink"><a class="biglink" href="numpy/user/">Numpy User Guide</a><br/>
-        <span><a href="numpy/numpy-user-1.17.0.pdf">[PDF]</a></span>
+        <span><a href="numpy/numpy-user.pdf">[PDF]</a></span>
       </p>
       <p class="biglink"><a class="biglink" href="numpy/f2py/">F2Py Guide</a><br/>
       </p>
       <p class="biglink"><a class="biglink" href="scipy/reference/">Scipy Reference Guide</a><br/>
-        <span><a href="scipy/scipy-html-1.7.1.zip">[HTML+zip]</a>,
-          <a href="scipy/scipy-ref-1.7.1.pdf">[PDF]</a></span>
+        <span><a href="scipy/scipy-html.zip">[HTML+zip]</a>,
+          <a href="scipy/scipy-ref.pdf">[PDF]</a></span>
```

Comment:
This does not look correct. `https://docs.scipy.org/doc/scipy/scipy-ref.pdf` returns 404